lib/commands/general.js

```diff
@@ -239,7 +240,7 @@ commands.reset = async function () {
     await this.adb.stopAndClear(this.opts.appPackage);
   }
   // reset context since we don't know what kind on context we will end up after app launch.
-  this.curContext = null;
+  this.curContext = NATIVE_WIN;
```

Comment:
what's the diff between NATIVE_WIN and NATIVE_APP?

Reply:
`NATIVE_WIN` is a constant with the value `'NATIVE_APP'` (see https://github.com/appium/appium-android-driver/blob/0a1727dea8d6a7e6d0acfd8c61f77882c9d57a85/lib/webview-helpers.js#L5).

Reply:
This changes introduce new issue with the following flow `Start Session -> switch to WebView -> Reset -> switch to Native -> Find Element` : `Selenium error: chrome not reachable`

Reply:
Updated to recreate the chrome session using the current logic.